### PR TITLE
Ports: Add stockfish port

### PR DESCRIPTION
**Pull request by tcl3**
This PR adds a port for the Stockfish chess engine.

Tested by running `stockfish bench` on the command line.

### CODE REVIEW

File: Ports/stockfish/package.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+auth_type='sha256'
+workdir="Stockfish-sf_${version}/src/"
+makeopts+=(ARCH="${SERENITY_ARCH}" SUPPORTED_ARCH=true COMPCXX="${CXX}")
+
+build() {
+    run make build "${makeopts[@]}" 
+}
+
+install() {
+    run cp stockfish "${SERENITY_INSTALL_ROOT}/usr/local/bin/"
```
Comment (LucasChollet) [inline Ports/stockfish/package.sh:15]: Please add:
`run mkdir -p ${SERENITY_INSTALL_ROOT}/usr/local/bin/`

**Review by LucasChollet — COMMENTED** [review]
Looking forward to play against it in `Chess`

File: Ports/stockfish/package.sh (hunk 1 above)
Comment (tcl3) [reply Ports/stockfish/package.sh:16]: Done :^)

**Comment by tcl3** [comment]
Latest push is a rebase to fix CI, no functional changes.

**Pull request merged**